src/components/players/player-others/players-others.tsx

```diff
@@ -75,24 +77,26 @@ export const PlayersOthers: FC<Props> = ({
       )}
       {stage === "results" && (
         <Stack sx={styles.results}>
-          <Typography variant="subtitle1" sx={styles.playerHeading}>
+          <Typography variant="h2" sx={styles.playerHeading}>
             {resultHeading}
           </Typography>
-          <Typography variant="body1" sx={styles.playerSubHeading}>
-            {resultText}
-          </Typography>
         </Stack>
       )}
       <Stack sx={styles.list(stage !== "chat")}>
         {otherPlayers.map((player, index) => {
+          const isSelected = localPlayer?.votes?.includes(player.userId);
```

Comment:
I don't think localPlayer can be undefined
```suggestion
          const isSelected = localPlayer.votes?.includes(player.userId);
```